data/getdata.sh

```diff
@@ -7,8 +7,8 @@ echo "Unzipping the data files (overwrites any existing .tsv files)..."
 gunzip -vf *.gz
 echo
 echo "Removing IMDb '\N' values..."
-sed --in-place 's/\\N//g' *.tsv
+ed -s *.tsv <<< 's/\\N//g'
 echo
 echo "IMDb data files ready:"
 ls *.tsv
-echo 
+echo
```

Comment:
```suggestion
echo
read -p "Press any key to continue... " -n1 -s
```

I think it's useful to have a wait statement, so that the user can read the messages on the screen, instead of having the window just disappear. Maybe one before as well, to make sure the user meant to have this script run.

I'm not sure if that's commonly done with shell scripts, but I think it would be useful for the user.